jbpm-flow/src/main/java/org/jbpm/workflow/instance/node/SubProcessNodeInstance.java

```diff
@@ -211,7 +212,7 @@ public void internalTrigger(final NodeInstance from, String type) {
                 List<String> businessKeys = new ArrayList<String>();
```

Comment:
Comment on the line 211 no longer holds (timestamp -> UUID).

Reply:
done